adam/curriculum/verbs_with_dynamic_prepositions_curriculum.py

```diff
@@ -545,6 +579,11 @@ def _make_go_with_prepositions(num_samples: int = 5, *, noise_objects: int = 0):
     goal_object_hollow = standard_object(
         "goal_object_hollow", required_properties=[HOLLOW]
     )
+    path_object = standard_object(
+        "path_object",
+        required_properties=[CAN_HAVE_THINGS_RESTING_ON_THEM, HAS_SPACE_UNDER],
```

Comment:
Nitpick: are the required properties here necessary for _in front of/behind_? My thought was to make a `path_object` for each of the two sets of PPs.